nifi-toolkit/nifi-toolkit-tls/src/main/java/org/apache/nifi/toolkit/tls/standalone/TlsToolkitStandalone.java

```diff
@@ -236,6 +236,6 @@ public void createNifiKeystoresAndTrustStores(StandaloneConfig standaloneConfig)
     }
 
     protected static String getClientDnFile(String clientDn) {
-        return clientDn.replace(',', '_').replace(' ', '_');
+        return clientDn.replace(',', '_').replace(' ', '_').replaceAll("[^\\w&&[^\\.]&&[^=]]", "_");
```

Comment:
Since the regular expression is being used in multiple places, I would recommend making this a `public static final` constant in one of the toolkit utility classes.

Reply:
This is also performing 3 different String replacement actions, which hurts performance. We should replace this with a single compiled `Pattern` using regex and perform the replacements at once.